<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkiSky</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button
{
background-color: transparent; 

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_bg_pure">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_2
{
background-color: transparent; 

}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_3
{
background-color: transparent; 

}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_4
{
background-color: transparent; 

}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="krynica_lab">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#krynica_lab
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>Krynica</string>
    </property>
   </widget>
   <widget class="QLabel" name="szczyrk_lab">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#szczyrk_lab
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>Szczyrk</string>
    </property>
   </widget>
   <widget class="QLabel" name="zieleniec_lab">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#zieleniec_lab
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>Zieleniec</string>
    </property>
   </widget>
   <widget class="QLabel" name="karpacz_lab">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#karpacz_lab
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>Karpacz</string>
    </property>
   </widget>
   <widget class="QLabel" name="karpacz_nieb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="karpacz_snieg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zieleniec_nieb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zieleniec_snieg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="szczyrk_nieb">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="szczyrk_snieg">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="krynica_nieb">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="krynica_snieg">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="szczyrk_stan">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="krynica_stan">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="karpacz_stan">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zieleniec_stan">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="karpacz_temp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#karpacz_temp
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zieleniec_temp">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#zieleniec_temp
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="szczyrk_temp">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#szczyrk_temp
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="krynica_temp">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#krynica_temp
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lc1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lc1
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lc2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lc2
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lc3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lc3
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lc4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lc4
{
background-color: white; 


border: none; 
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <zorder>label_bg_pure</zorder>
   <zorder>zieleniec_stan</zorder>
   <zorder>szczyrk_stan</zorder>
   <zorder>krynica_stan</zorder>
   <zorder>karpacz_stan</zorder>
   <zorder>button</zorder>
   <zorder>button_2</zorder>
   <zorder>button_3</zorder>
   <zorder>krynica_lab</zorder>
   <zorder>szczyrk_lab</zorder>
   <zorder>zieleniec_lab</zorder>
   <zorder>karpacz_lab</zorder>
   <zorder>karpacz_nieb</zorder>
   <zorder>karpacz_snieg</zorder>
   <zorder>zieleniec_nieb</zorder>
   <zorder>zieleniec_snieg</zorder>
   <zorder>szczyrk_nieb</zorder>
   <zorder>szczyrk_snieg</zorder>
   <zorder>krynica_nieb</zorder>
   <zorder>krynica_snieg</zorder>
   <zorder>karpacz_temp</zorder>
   <zorder>zieleniec_temp</zorder>
   <zorder>szczyrk_temp</zorder>
   <zorder>krynica_temp</zorder>
   <zorder>button_4</zorder>
   <zorder>lc1</zorder>
   <zorder>lc2</zorder>
   <zorder>lc3</zorder>
   <zorder>lc4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAktualizacja_Danych"/>
    <addaction name="actionZmiana_widoku_mapy"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWyb_r_Szczytu">
   <property name="text">
    <string>Wybór Szczytu</string>
   </property>
  </action>
  <action name="actionAktualizacja_Danych">
   <property name="text">
    <string>Aktualizacja Danych</string>
   </property>
  </action>
  <action name="actionZmiana_widoku_mapy">
   <property name="text">
    <string>Zmiana widoku mapy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
